.github/workflows/build-master.yml

```diff
@@ -33,12 +33,6 @@ jobs:
                     name: Ballerina Internal Log
                     path: graphql-ballerina/ballerina-internal.log
                     if-no-files-found: ignore
-            -   name: Archive Code Coverage JSON
```

Comment:
Do we need to remove this? IMO, keeping this will help developers get the exact lines for the coverage since we would not produce that info with jacoco report.

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-graphql/pull/52/commits/3aefd771a665a691be83bf184e9c052e1218c02b